fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -447,7 +439,7 @@ private RdfStream getTriples(final FedoraResource resource, final TripleCategory
 
     protected URI getUri(final FedoraResource resource) {
         try {
-            final String uri = translator().reverse().convert(resource).getURI();
```

Comment:
As a separate issue, should we go ahead and purge all usage of `translator()`?

Reply:
I think this might be more easily tackled once we can get a response from the server. But I'm not opposed if @dbernstein feels he can fix them all, it will make this a more complex PR.

Reply:
Defer to JIRA:  https://jira.lyrasis.org/browse/FCREPO-3163

Reply:
I created a JIRA: https://jira.lyrasis.org/browse/FCREPO-3163

Reply:
I'm good with this, @awoods can we consider this *resolved*?